### ISSUE TITLE
Request to integrate Mesh for Data with ODH

### ISSUE THREAD

**Issue by eletonia** [operate-first/odh-moc-support#30]
Hello, 

[IBM Mesh for Data](https://github.com/IBM/the-mesh-for-data)(M4D) is interested in integrating with OpenDataHub. After speaking with @durandom and @tumido , we agree that M4D would be a great extension of features providing data cataloguing and data governance. We hope to specifically integrate with JupyterHub and Elyra in ODH. 

This issue will act as a placeholder and means to track / reference.

**Comment by tumido** [comment]
cc @pacospace for visiblity (I know you're into all Elyra related things around here!)

**Comment by pacospace** [comment]
> cc @pacospace for visiblity (I know you're into all Elyra related things around here!)

Thanks @tumido! We are working on the extension there as well indeed :) https://www.youtube.com/watch?v=IBzTOP4TCdA

**Comment by durandom** [comment]
@eletonia I think the first step would be to deploy a "hello world" of M4D on the MOC cluster and see how it integrates with the components of ODH deployed there.

Do you have a corresponding issue at https://github.com/IBM/the-mesh-for-data to track progress and what the "hello world" would look like?

**Comment by eletonia** [comment]
@durandom Yes I just created a corresponding issue [here](https://github.com/IBM/the-mesh-for-data/discussions/259) 
Once we have access to the MOC cluster, we will try deploying there.
